Makefile

```diff
@@ -219,24 +216,22 @@ CONTAINER_CMD:=docker run --rm \
 		quay.io/coreos/jsonnet-ci
 
 .PHONY: generate
-generate: jsonnet-vendor ${MANIFESTS} README.md
+generate: jsonnet-vendor examples/manifests README.md
 
 .PHONY: generate-in-docker
 generate-in-docker:
 	@echo ">> Compiling assets and generating Kubernetes manifests"
 	$(CONTAINER_CMD) make $(MFLAGS) generate
 
-.PHONY: ${MANIFESTS}
-${MANIFESTS}: jsonnet/main.jsonnet jsonnet/lib/* | $(JSONNET) $(GOJSONTOYAML)
-	@rm -rf ${MANIFESTS}
-	@mkdir -p ${MANIFESTS}
-	$(JSONNET) -J jsonnet/vendor -m ${MANIFESTS} jsonnet/main.jsonnet | xargs -I{} sh -c 'cat {} | $(GOJSONTOYAML) > {}.yaml && rm -f {}' -- {}
+examples/manifests: examples/main.jsonnet examples/vendor jsonnet/lib/* | $(JSONNET) $(GOJSONTOYAML)
+	@rm -rf examples/manifests
+	@mkdir -p examples/manifests
+	$(JSONNET) -J examples/vendor -m examples/manifests examples/main.jsonnet | xargs -I{} sh -c 'cat {} | $(GOJSONTOYAML) > {}.yaml && rm -f {}' -- {}
 
-.PHONY: jsonnet-vendor
```

Comment:
CI complains with:
```
make: *** No rule to make target 'jsonnet-vendor', needed by 'generate'.  Stop.
```
If we remove this, then we need to update the circle CI config too

Reply:
Yeah, looking into this already. Sorry, I should have commented, I guess. :innocent: